package.json

```diff
@@ -53,7 +53,7 @@
     "hooks": {
       "post-merge": "sh scripts/dependencyUpdateCheck.sh",
       "pre-commit": "npm run test:lint",
-      "pre-push": "sh scripts/stopPushOnLatest.sh && sh scripts/upToDateWithLatest.sh && sh scripts/enforceVersions.sh && npm run test:unit && npm run snyk"
+      "pre-push": "sh scripts/stopPushOnLatest.sh && sh scripts/upToDateWithLatest.sh && sh scripts/enforceVersions.sh && npm run test:unit && npm run snyk && npx apache2-license-checker"
```

Comment:
Im not sure this is doing what we want. doesnt `npx apache2-license-checker` just run https://www.npmjs.com/package/apache2-license-checker not the bbc one?

Reply:
`npx @bbc/apache2-license-checker` might be what we want? Also if you use npx you dont need to put it in your package json as a dependancy.

Reply:
Oooooh good spot!

Reply:
I recently decided to remove the security flaw in my bashrc setup and forgo the convenience of a `export PATH=$PATH:./node_modules/.bin`. This meant that commands that used to work for me (`apache2-license-checker`) now errored (`apache2-license-checker: command not found`).

Using `npx` runs the package without having to specify the gross `node_modules/` prefix. I'd advise against downloading it each time because we tend to push code a lot and it would slow us down to have to download `apache2-license-checker` every time we push.